test/fixtures/es-modules/tla/unresolved-with-worker-process-exit.mjs

```diff
@@ -0,0 +1,10 @@
+import { Worker } from 'worker_threads';
+
+// Do not use isMainThread so that this test itself can be run inside a Worker.
+if (!process.env.HAS_STARTED_WORKER) {
+  process.env.HAS_STARTED_WORKER = 1;
```

Comment:
I think we could use `isMainThread` here since we are not running this test from a worker.

Reply:
done